cirq-core/cirq/ops/identity_test.py

```diff
@@ -208,3 +208,8 @@ def test_identity_short_circuits_act_on():
     args = mock.Mock(cirq.SimulationState)
     args._act_on_fallback_.side_effect = mock.Mock(side_effect=Exception('No!'))
     cirq.act_on(cirq.IdentityGate(1)(cirq.LineQubit(0)), args)
+
+
+def test_identity_commutes():
+    assert cirq.I._commutes_(cirq.X)
+    assert cirq.I._commutes_("placeholder") is NotImplemented
```

Comment:
It is better to check this works as intended in the API function -

```suggestion
    assert cirq.commutes(cirq.I, cirq.X)
    with pytest.raises(TypeError):
        cirq.commutes(cirq.I, "Gate")
```